<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEditorClass</class>
 <widget class="QMainWindow" name="MapEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MapEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>0</y>
      <width>100</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRemove">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>700</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditX">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>530</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelObjName">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>500</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>未选中</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>530</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>横坐标</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>560</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>纵坐标</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditY">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>560</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>590</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>方向</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDirect">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>590</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSpecialSettingName">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>610</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>特殊设置</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSA">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>640</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSync">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>660</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>同步</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>组件</string>
    </property>
    <addaction name="actionPartRollingWall"/>
    <addaction name="actionPartWall"/>
    <addaction name="actionBuff"/>
    <addaction name="actionTankSpawn"/>
    <addaction name="actionShortWall"/>
    <addaction name="actionMidWall"/>
    <addaction name="actionMovableBlock"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>地图</string>
    </property>
    <addaction name="actionReadFromFile"/>
    <addaction name="actionReadFromClipboard"/>
    <addaction name="separator"/>
    <addaction name="actionSaveToFile"/>
    <addaction name="actionCopyToClipboard"/>
    <addaction name="separator"/>
    <addaction name="actionRandomMap"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <action name="actionPartRollingWall">
   <property name="text">
    <string>RollingWall</string>
   </property>
  </action>
  <action name="actionPartWall">
   <property name="text">
    <string>Wall</string>
   </property>
  </action>
  <action name="actionBuff">
   <property name="text">
    <string>Buff</string>
   </property>
  </action>
  <action name="actionTankSpawn">
   <property name="text">
    <string>TankSpawn</string>
   </property>
  </action>
  <action name="actionReadFromFile">
   <property name="text">
    <string>从文件读取</string>
   </property>
  </action>
  <action name="actionReadFromClipboard">
   <property name="text">
    <string>从剪贴板读取</string>
   </property>
  </action>
  <action name="actionSaveToFile">
   <property name="text">
    <string>保存到文件</string>
   </property>
  </action>
  <action name="actionCopyToClipboard">
   <property name="text">
    <string>复制到剪贴板</string>
   </property>
  </action>
  <action name="actionRandomMap">
   <property name="text">
    <string>随机生成一张地图</string>
   </property>
  </action>
  <action name="actionShortWall">
   <property name="text">
    <string>ShortWall</string>
   </property>
  </action>
  <action name="actionMidWall">
   <property name="text">
    <string>MidWall</string>
   </property>
  </action>
  <action name="actionMovableBlock">
   <property name="text">
    <string>MovableBlock</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
